### PR TITLE
Thundra python agent GitHub action

### PR DESCRIPTION
**Pull request by gokhan721**

### CODE REVIEW

**Review by rwxdash — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,97 @@
...
+        run: |
+            pip install pytest
+            pip install -r requirements.txt
+
+      - name: Install Thundra Python Agent
+        uses: thundra-io/thundra-foresight-python-action@v1
+        with:
+          apikey: ${{ secrets.THUNDRA_APIKEY }}
+          project_id: ${{ secrets.THUNDRA_PROJECT_ID }}
+          command: pytest ${TEST_FOLDER}
```
Comment (rwxdash) [inline README.md:48]: Let's remove `${TEST_FOLDER}` from here.

File: README.md [hunk 2]
```diff
@@ -1 +1,97 @@
...
+```yaml
+# ...
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    
+    steps:
+      
+      => Predefined steps for setting up and installing both system and test dependencies
```
Comment (rwxdash) [inline README.md:65]: Remove this and write the basic steps like the previous example.

File: README.md [hunk 3]
```diff
@@ -1 +1,97 @@
...
+
+      - name: Install Thundra Python Agent
+        uses: thundra-io/thundra-foresight-python-action@v1
+        with:
+          apikey: ${{ secrets.THUNDRA_APIKEY }}
+          project_id: ${{ secrets.THUNDRA_PROJECT_ID }}
+
+      => Test Steps
```
Comment (rwxdash) [inline README.md:73]: Remove this and write the basic steps like the previous example. We can just add a step with `run: pytest` or whatever the right way is.

**Review by rwxdash — APPROVED** [review]

File: src/actions/helper/command.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (bcaglayan) [inline src/actions/helper/command.ts:1]: is this helper file used in this project ?

**Review by suleymanbarman — APPROVED** [review]

**Pull request merged**